Makefile

```diff
@@ -37,6 +38,12 @@ lint:  ## Run linter
 lint: $(GOLANGCI_LINT)
 	$(GOLANGCI_LINT) run -c .golangci.yml
 
+.PHONY: gofumpt
+$(GOFUMPT): install/deps
+gofumpt:  ## Run formatter
+gofumpt: $(GOFUMPT)
+	$(GOFUMPT) -w -l .
```

Comment:
One minor change - can the name of the make target be `format:` instead of `gofumpt:` thanks!

Reply:
Changed `gofumpt` to `format`. Thanks!